docs/src/interface.md

```diff
@@ -0,0 +1,66 @@
+# InteratomicPotentials Interface
+
+<!--
+TODO (Dallas): write explanation of how to create new potentials of various types
+example: https://cesmix-mit.github.io/Atomistic.jl/dev/interface/
+-->
+
+# Instantiating a Built-In Interatomic Potentials
+There are a number of built-in interatomic potentials that are available for users: Born-Mayer-Huggins, Lennard-Jones, Coulomb's Law, and Morse. These potentials are subtypes of `EmpiricalPotential`. There are additional basis potentials, the Spectral Neighbor Analysis Potential and Atomic Cluster Expansion, available in the related package `InteratomicBasisPotentials.jl`. In order to instantiate a potential, you need to provide the necessary potential parameters (see API documentation), radial cutoff, and species (i.e. elements) for which the potential is defined for.
+
+Below is an example for the Lennard-Jones potential:
+```julia
+ϵ = 0.1034 * u"eV"
+σ = 1.0 * u"Å"
+rcutoff = 4.0 * σ
+species = [:Ar, ]
+lj = LennardJones(ϵ, σ, rcutoff, species)
+```
+
+In general, empirical potentials handles have the signature:
+```julia
+MockPotential{T<:AbstractFloat} <: EmpiricalPotential{NamedTuple{(:a, :b, ..., :e)},NamedTuple{(:rcutoff,)}}
```

Comment:
I would make your example `CustomPotential` instead of `MockPotential{T<:AbstractFloat}`